benchmark/tinysegmenter3.py

```diff
@@ -21,16 +22,16 @@
 
 __all__ = ["tokenize"]
 
-_patterns = {
-    "[一二三四五六七八九十百千万億兆]":"M",
-    "[一-龠々〆ヵヶ]":"H",
-    "[ぁ-ん]":"I",
-    "[ァ-ヴーｱ-ﾝﾞｰ]":"K",
-    "[a-zA-Zａ-ｚＡ-Ｚ]":"A",
-    "[0-9０-９]":"N"
-}
+_patterns = [
+    ("[一二三四五六七八九十百千万億兆]", "M"),
+    ("[一-龠々〆ヵヶ]", "H"),
+    ("[ぁ-ん]", "I"),
+    ("[ァ-ヴーｱ-ﾝﾞｰ]", "K"),
+    ("[a-zA-Zａ-ｚＡ-Ｚ]", "A"),
+    ("[0-9０-９]", "N"),
+]
```

Comment:
In julia code,  "H" flag is caught by L67 and L86
https://github.com/chezou/TinySegmenter.jl/blob/master/src/TinySegmenter.jl#L86

So, this python code is not wrong.